docs/docs/clients/overview.md

```diff
@@ -120,34 +121,76 @@ running in `Local Evaluation` mode.
+- If the context of an Identity changes, for example if a new Trait is added to the Identity, the SDK will make a new
+  call to the Flagsmith API to update the Traits of the Identity and receive new Flags, as the value of those flags may
+  have changed. This happens in one network call.
+- If an entirely new Identity is provided, the SDK will make a new call to the Flagsmith API to receive new Flags.
+- Other than the above two points, the SDK will not make further network requests to the Flagsmith API. If you wish you
+  can manually trigger a network call in code and refresh Flags locally.
 
-The benefit of running in Local Evaluation mode is that you can process flag evaluations much more efficiently as they
-are all computed locally.
+#### Server-side
 
-:::tip
+- Flagsmith SDKs do not store Flags in local memory. Every Flag evaluation in your code will trigger a network request.
 
-In circumstances where you need to target a specific identity, you can do this by creating a segment to target that
-specific user and subsequently adding a segment override for that segment.
+If this approach does not work for you (generally for reasons of latency or overly chatty networking) you should
+consider Local Evaluation mode (explained below) or the [Edge Proxy](/advanced-use/edge-proxy).
 
-:::
+### Local Evaluation
+
+Local Evaluation mode is only available with Server-side SDKs.
+
+- When the SDK is initialised, it will make a network request to the Flagsmith API to receive all the Environment data
+  it needs to run Local Evaluations. The SDK receives a single JSON document with all this data.
+- Future evaluations are all computed locally within the SDK runtime. This means they are extremely fast as there is no
+  network latency to account for.
+- No further network calls take place for 60 seconds.
+- After 60 seconds have elapsed, the SDK will refresh the JSON Environment document with a network call to the Flagsmith
+  API.
+
+### Client-side SDK approaches
+
+Since Client-side SDKs will generally be associated with a single Identity (the person who owns the client device!), a
+common pattern for networking implementation is:
+
+1. The user launches your application for the first time. Since your application does not know how they are, a call to
+   get the default Environment Flags is made by the Flagsmith SDK.
+2. After the user logs in, make a second request to get the Flags with the new user Identity.
+3. Cache these flags locally on the device.
+4. Upon subsequent application launches, immediately read the flags from the cached store from step 3 and use those as
+   your application Flag values.
+5. In the background make a request to Flagsmith to refresh their Flags.
+6. Update both your application and the local cache with these fresh flags.
+
+### Things to note
 
-## Local Evaluation Runtime Considerations
+- Flagsmith always returns **all** Environment Flags during API calls. In fact, there is no way to request the state of
```

Comment:
>In fact, there is no way to request the state of a single flag via our API.

I don't think this is _technically_ true - it is possible via the API we just don't advertise it (because it's poorly implemented). Happy to keep this here though, just clarifying.